src/docs/release-notes/R21/R21.mdx

```diff
@@ -0,0 +1,73 @@
+-   All icons (Material Design and PX Blue icons) are now searchable by their aliases in Figma.
+-   Added more system components, including: phone keyboard, phone keypad, status bar, browser window, cursor, system dialog box and time picker.
+-   Added table components.
+
+### Themes
+
+-   Added a new `stroked` Angular Material chip style variant.
+-   Enhanced showcase demos for [Angular](https://github.com/pxblue/angular-showcase-demo/tree/master) and [React](https://github.com/pxblue/react-showcase-demo) to display all PX Blue and Material components in an organized manner.
+-   Simplified [Dark Theming for React Native](https://github.com/pxblue/react-native-themes/tree/master#dark-theme) (eliminated the need for alternate dark theme and provide special pre-themed wrappers for React Native Paper components).
+
+### Documentation Site
+
+-   Updated Angular and React design pattern code examples for: [Account Menu](/patterns/account-menu), [App Bar](/patterns/appbar), [Forms & Validations](/patterns/forms) and [Loading & Waiting](/patterns/loading).
+
+### Re-usable Authentication Workflows
+
+-   Angular, React and React Native authentication workflows' translations are now synced up.
+-   In all three authentication workflows, users can now go back and adjust their previous inputs during registration.
+-   Reordered screens in the self-registration process so accepting the EULA appears as the first step.
+
+### PX Blue CLI
+
+-   Updated to use the latest dependencies.
+
+### Miscellaneous
+
+-   Bug fixes & minor updates.
+-   Added copyright information to all our package source code.
+-   Reorganized [icons](https://github.com/pxblue/icons) and [themes](https://github.com/pxblue/themes) repository.
+-   Improved code coverage with unit testing and Cypress.
+
+### Package Summary
+
+#### Angular / Ionic
+
+-   @pxblue/angular-components v5.0.1, v6.0.0 ([Changelog](https://github.com/pxblue/angular-component-library/releases))
+-   @pxblue/angular-themes v6.3.0 ([Changelog](https://github.com/pxblue/angular-themes/releases))
+-   @pxblue/angular-auth-workflow v2.4.0 ([Changelog](https://github.com/pxblue/angular-workflows/releases))
+-   @pxblue/icons v1.7.0 ([Changelog](https://github.com/pxblue/icons/releases))
+-   @pxblue/icons-svg v1.7.0 ([Changelog](https://github.com/pxblue/icons/releases))
+-   @pxblue/ng-progress-icons v1.2.1 ([Changelog](https://github.com/pxblue/progress-icons/releases))
+
+#### React
+
+-   @pxblue/react-components v5.3.1 ([Changelog](https://github.com/pxblue/react-component-library/releases))
+-   @pxblue/react-auth-workflow v2.4.0 ([Changelog](https://github.com/pxblue/react-workflows/releases))
+-   @pxblue/react-auth-shared v3.5.0 ([Changelog](https://github.com/pxblue/react-auth-shared/releases))
```

Comment:
Meant to also say this is a behind-the-scenes package that users don't really know about, so I would leave it off this list (and off the RN one below)

Reply:
So you think nobody would want to refer to this list and update their existing auth workflow packages in their projects?

Reply:
They would never need to update the react-auth-shared package separately. It doesn't even show up in their project dependency array in package.json (they just install the react-auth-workflow or react-native-auth-workflow package and pick up the necessary version of shared-auth through its child dependencies). Basically, any updates that we make in the shared-auth package would be caused by something we are changing in one of the workflow packages and we would just tell users to update to the newest version of the workflow package (which includes the shared-auth updates).

Does that make sense? 

I think 'revealing' the package behind the curtain will just confuse them and make them think they need to track the two packages (shared-auth and workflow) separately when they do not.